README.md

```diff
@@ -147,9 +147,34 @@ _Note: It is recommended you migrate to using the `pe_r10k` module which is basi
 a clone of this modules features and file tickets for anything missing._
 
 
-## Usage
+### Using a internal gem server
 
-Installing using a proxy server
+```puppet
+# Create a global gemrc for Puppet Enterprise to add the local gem source
+# See http://projects.puppetlabs.com/issues/18053#note-12 for more information.
+
```

Comment:
why not leave this as text? rather than code

Reply:
why? copy&paste.

fixed it anyways.